airbyte-integrations/connectors/destination-typesense/destination_typesense/destination.py

```diff
@@ -14,11 +14,15 @@
 
 
 def get_client(config: Mapping[str, Any]) -> Client:
-    node = {"host": config.get("host"), "port": config.get("port") or "8108", "protocol": config.get("protocol") or "https"}
+    hosts = config.get("host").split(",")
     path = config.get("path")
-    if path:
-        node["path"] = path
-    client = Client({"api_key": config.get("api_key"), "nodes": [node], "connection_timeout_seconds": 3600})
+    nodes = []
+    for host in hosts:
+        node = {"host": host, "port": config.get("port") or "8108", "protocol": config.get("protocol") or "https"}
```

Comment:
You're considering if they user have different servers/host they're using the same port and protocol. Is this correct?

Reply:
Correct. This is the way I found to allow connection without impacting what exists today.

I think this approach will meet most cases. You could keep the same port and protocol unless you run more than one instance on the same machine.